boards/arduino-leonardo/src/lib.rs

```diff
@@ -172,20 +177,23 @@ pub mod pwm {
     pub use atmega32u4_hal::pwm::*;
 }
 
+pub type Baudrate = hal::usart::Baudrate<hal::clock::MHz16>;
+
 /// Serial (UART) interface on pins `D0` (RX) and `D1` (TX)
 ///
 /// # Example
 /// For a full example, see [`examples/leonardo-serial.rs`][ex-serial].  In short:
 /// ```no_run
 /// let dp = arduino_leonardo::Peripherals::take().unwrap();
 ///
+/// let baudrate = 57600.into_baudrate;
 /// let mut pins = arduino_leonardo::Pins::new(dp.PORTB, dp.PORTC, dp.PORTD, dp.PORTE);
 ///
 /// let mut serial = arduino_leonardo::Serial::new(
 ///     dp.USART1,
 ///     pins.d0,
 ///     pins.d1.into_output(&mut pins.ddr),
-///     57600,
+///     baudrate,
```

Comment:
Don't create an extra variable here:

```suggestion
/// let mut pins = arduino_leonardo::Pins::new(dp.PORTB, dp.PORTC, dp.PORTD, dp.PORTE);
///
/// let mut serial = arduino_leonardo::Serial::new(
///     dp.USART1,
///     pins.d0,
///     pins.d1.into_output(&mut pins.ddr),
///     57600.into_baudrate(),
```